.github/workflows/project-codeflare-release.yml

```diff
@@ -165,3 +165,29 @@ jobs:
           echo "Kueue release with version ${{ github.event.inputs.kueue-version }} does not exist. Please select an existing version."
           exit 1
         fi
+
+  generate-component-metadata:
+    needs: [check-appwrapper-version, check-kuberay-version, check-kueue-version]
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Generate component_metadata.yaml
+      run: |
+        cat <<EOL > config/component_metadata.yaml
+        releases:
```

Comment:
As far as I know we should just include the components that are referenced in the DSC resource, so codeflare, ray, kueue and training operator. We need to update separate files in separate repos for each.
We need to add training operator but as it is not GA we don't want to include it in the stack compatibility matrix in the readme yet. 
I wonder if a separate workflow that is called in the release workflow would work better.